angular-chart.js

```diff
@@ -152,7 +159,7 @@
           scope.$watch('series', resetChart, true);
           scope.$watch('labels', resetChart, true);
           scope.$watch('options', resetChart, true);
-          scope.$watch('colours', resetChart, true);
+          setColoursListener();
```

Comment:
I could do the same for the other attributes, but I noticed the only attribute being modified in the $watch (and being watched) is the colours attribute. I wanted to keep the changes down to a minimum.